AcademyApp/Designables/UIButtonX.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIButtonX.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 14/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class UIButtonX: UIButton {
+
+    @IBInspectable
+    var cornerRadius : CGFloat = 0 {
+        didSet {
+            self.layer.cornerRadius = cornerRadius
+        }
+    }
+    
+    var disabledColor = UIColor.greyishBrown
+    var enabledColor = UIColor.pinkishRed
+    
```

Comment:
disableColor and enabledColor don't have to be publicly known

Reply:
Updated properities to be private.